test/common_test.py

```diff
@@ -424,3 +439,57 @@ def test_get_set_invalid_args(
     get_method = getattr(set_api_client, f"get_{set_type}_set_v1")
     with pytest.raises(ValueError, match=include_params_valid(inc_arg)):
         get_method(context, {"ref": KBASE_UPA, inc_arg: 666})
+
+
+def check_get_set(
```

Comment:
these are all the preparatory checks done to create the parameters for a `get_***_set_v1` query